docs/how-to-build-a-lexical-model.md

```diff
@@ -0,0 +1,365 @@
+`Latn`) and in Unified Canadian Aboriginal Syllabics ([ISO 15924][]:
+`Cans`). Say my model is only designed for syllabics. The correct BCP 47 tag
+is:
+
+    crk-Cans
+
+However, to create a valid lexical model ID, we need to replace the
+dash with an underscores, and convert all uppercase letters to
+lowercase. Thus, our `bcp47` is:
+
+    crk_cans
+
+[ISO 639-3]: https://iso639-3.sil.org/code_tables/639/data
+[ISO 15924]: https://en.wikipedia.org/wiki/ISO_15924
+[BCP 47]: https://tools.ietf.org/html/bcp47
+
+### Uniq
+
+_uniq_ is an arbitrary tag that uniquely identifies your specific model.
+This identifier can be anything, as long as it hasn't already been used
+before for this specific language. As with `author`
+identifier is one or more ASCII lowercase letters, digits, and the underscore
+(`_`), and it must start with a lowercase ASCII letter.
+
+### Example
+
+My author is called `example`, and the BCP 47 for my language is `str`.
+I'll give my model the unique tag of `tutorial`. Thus my unique model
+is:
+
+    example.str.tutorial
+
+
+Step 4: Creating the package files
+----------------------------------
+
+Now that we have a unique ID for our model, and wordlist data, we can
+the appropriate packaging files.
+
+Say we're currently in a folder called `release`: our model will follow
+this template for its directory structure.
+
+```
+release/
+└── {author}/
+    └── {bcp47}.{uniq}/
+        ├── {author}.{bcp47}.{uniq}.model_info
+        └── source/
+            ├── {author}.{bcp47}.{uniq}.model.kps
+            ├── wordlist.tsv
+            └── model.ts
+```
+
+Notice how under `release`, there is a folder that is named after the
+author. Within that folder is a folder for each models created by that
+particular author. Within each model folder, this is a folder called
+`source` that contains **wordlist.tsv**. We will discuss what the other
+files are called a bit later in this document.
+
+For our example of `example.str.tutorial`, I'll want the following
+folder structure:
+
+```
+release/
+└── example/
+    └── str.tutorial/
+        ├── example.str.tutorial.model_info
+        └── source/
+            ├── example.str.tutorial.model.kps
+            ├── wordlist.tsv
+            └── model.ts
+```
+
+To create this folder structure, I can execute the following in
+Git Bash or the terminal:
+
+    cd /type/the/path/to/release
+    mkdir -p example/str.tutorial/source
+
+Now move the wordlist to the appropriate place:
+
+    mv /type/the/current/path/to/wordlist.tsv example/str.tutorial/source/
+
+I now have the following folder structure:
+
+```
+.
+└── example
+    └── str.tutorial
+        └── source
+            └── wordlist.tsv
+```
+
+We need to create the remaining files:
+ - The "model info" file: `example.str.tutorial.model_info`
+ - The package file: `example.str.tutorial.model.kps`
+ - The model configuration file: `model.ts`
+
+
+Step 4.1: Creating the model info file
```

Comment:
model_info should only be required when building within the repo and distributing via keyman cloud (not used for private, ad-hoc distribution; worth noting this). We'll be adding outside-repo builds before release, I hope!